pkg/garden/app/docket.hoon

```diff
@@ -123,6 +123,17 @@
         [%charges ~]
       ?>  (team:title [our src]:bowl)
       `state
+    ::
+        [%glob @ ~]
+      ~&  path+path
+      =/  desk  (~(got by by-base) i.t.path)
```

Comment:
might want to add a ~| around the got:by so that crashes will be easier to debug -- same with charges on the next line